Bottom: 6970bf57515be3b0b219e69b27ade6e2352a6b45
Top:    c3f75872adb13d5336939bfbdeaddb018037a80f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 21:02:25 +0000

Refresh of limit-virtual-memory-size-of

---

diff --git a/src/polygon.c b/src/polygon.c
index af2cc26..26e0cf9 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -136,7 +136,7 @@ polygon_init (void)
 
   /* DEBUG - AVOID PCB running the system out of memory! */
   getrlimit (RLIMIT_AS, &limit);
-  limit.rlim_cur = MIN (limit.rlim_cur, 7000 * 1024 * 1024 /* 7000 GiB limit to virtual memory size */);
+  limit.rlim_cur = MIN (limit.rlim_cur, 2000 * 1024 * 1024 /* 2 GiB limit to virtual memory size */);
   setrlimit (RLIMIT_AS, &limit);
 
 }
